SherpaBackEnd/Template/Infrastructure/Persistence/templates.json

```diff
@@ -2637,281 +2637,5 @@
         "position": 70
       }
     ]
-  },
-  {
```

Comment:
Are this supposed to be gone?

Reply:
yes, this was a template that I was trying to use have the option for a short survey, but at the moment it's not possible to have two different hackman surveys in the app